site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -92,61 +124,43 @@ longitude DOUBLE,
 zip_code varchar(30),
 time_zone varchar(8)
 );
+
 --Create a file format for the file
 CREATE OR REPLACE FILE FORMAT LOCATION_CSV
 SKIP_HEADER = 1
-FIELD_OPTIONALLY_ENCLOSED_BY = '"'
+FIELD_OPTIONALLY_ENCLOSED_BY = '\"'
 COMPRESSION = AUTO;
+
 --create a stage so we can upload the file
 CREATE STAGE LOCATION_DATA_STAGE
-file_format = LOCATION_CSV;
-```
-
-Your database should now look like the following
-
-<img src="assets/producer_org.png" width="425" />
+file_format = LOCATION_CSV;"
 
+--Go ahead now and upload the file to the stage.  After the file is uploaded go back to
+your worksheet and copy the file into the table created earlier
 
-You now need to upload the file you just downloaded from IP2Location to the stage you just created in Snowflake.  There are a couple of ways to do this:
+snow object stage copy /USER_PATH_HERE/IP2LOCATION-LITE-DB11.CSV @location_data_stage --database ip2location --schema ip2location 
 
-* Using Snowsight
-* Using SnowSQL
-
-Using Snowsight, come out of Worksheets and go to **data/databases** and then navigate to the stage we just created.  You should see a screen similar to the below
-
-<img src="assets/location_data_upload.png" width="800" />
-
-Go ahead now and upload the file to the stage.  After the file is uploaded go back to your worksheet and copy the file into the table created earlier
-
-```sql
---Copy the file into the table we just created
-COPY INTO LITEDB11 FROM @LOCATION_DATA_STAGE;
+snow sql -q "copy into litedb11 from @location_data_stage;
+SELECT COUNT(*) FROM LITEDB11;SELECT * FROM LITEDB11 LIMIT 10;" --database ip2location --schema ip2location
 ```
 
-With the data loaded you can run some test queries to get a feel for the data
-
-```sql
-SELECT COUNT(*) FROM LITEDB11;
-SELECT * FROM LITEDB11 LIMIT 10;
-```
 We now have the reference database setup in the provider account so are now ready to start building the application itself.
 
 <!-- ------------------------ -->
 ## Provider Setup
 Duration: 5
 
-Staying in the same worksheet we will carry on creating the application.  As part of the application build we will need to upload some files to a stage again.  You can either clear out the stage you just used, or you can create another stage.  Here we create another stage
+> aside positive
+> 
+> **Note** - The code is here to illustrate how would you normally do the app creation using the manual steps, but this step is executed automatically by the CLI, for example, in a manual environment you will need to update your files to a stage, but that managed by the **`snow app run`** command.  In the of the following steps there will be instructions on how to execute it.
```

Comment:
It's not clear to me that we don't need to create the stage / application package, but we _do_ need to do the reference_usage and sharing stuff. Was that already moved over to a package script?